src/hyperlight_host/src/sandbox/config.rs

```diff
@@ -236,6 +262,8 @@ impl Default for SandboxConfiguration {
             Self::INTERRUPT_VCPU_SIGRTMIN_OFFSET,
             #[cfg(gdb)]
             None,
+            #[cfg(crashdump)]
+            true,
```

Comment:
Do we want `true` or `false` to be the default when the feature is enabled? I'm leaning towards `false` but unsure